examples/simple_index.html

```diff
@@ -13,10 +13,13 @@ <h1>Nothing to see here, open a debug console in your browser to see the interes
 
     <script>
       // Input string (English and German mixed).
-      let input_value = 'Mo-Fr 10:00-20:00; Samstag und Sonntag geschlossen';
+      let input_value = 'Mo-Tu 10:00-20:00; Mi-Fr 10:00-20:00; Samstag und Sonntag geschlossen';
```

Comment:
Why change this example? In case you want to change it, prefer this one:

```
Mo-Tu,Mi-Fr 10:00-20:00; Samstag und Sonntag geschlossen
```

It avoids the duplicate (same) time range which helps with maintaining valid data. I just want to be careful what example I give because people might use them as role model.

Reply:
Yes, the intention was to have an 'inpure' example with german and english parts (as it is said in the comment a line before). Your suggested one is actually better, I've changed the PR to this :-)